### ISSUE TITLE
ShuffleSplit

### ISSUE THREAD

**Issue by arembridge** [udacity/machine-learning#347]
https://github.com/udacity/machine-learning/blob/master/projects/boston_housing/boston_housing.ipynb

The below line of code in udacity/machine-learning/boston_housing.ipynb failed for me:

* from sklearn.cross_validation import ShuffleSplit

I believe it should be:

* from sklearn.model_selection import ShuffleSplit

This is also the case in https://github.com/udacity/machine-learning/blob/master/projects/visuals.py.  I believe the following line:

* from sklearn.cross_validation import ShuffleSplit, train_test_split

should be 

* from sklearn.model_selection import ShuffleSplit, train_test_split

**Comment by singhjagmohan1000** [comment]
@arembridge Is code failed or it gave Deprecation warning? `sklearn.cross_validation` was deprecated in version 0.18 and as you said new version has `sklearn.model_selection`.

**Comment by tanujdhiman** [comment]
Exactly sklearn.cross_validation was deprecated. Work with sklearn.model_selection.

**Comment by Ashishkumarpanda** [comment]
Sklearn.cross_validation was depricated
Try using sklearn.model_selection

**Comment by ronny-udacity** [comment]
Starting from sklearn v0.20, the package `sklearn.cross_validation` is now deprecated. If you're using sklearn's newer version, please change this line of code:
- from: `from sklearn.cross_validation import ShuffleSplit `
- to: `from sklearn.model_selection import ShuffleSplit `
